guides/release/getting-started/overview-of-an-ember-app.md

```diff
@@ -0,0 +1,130 @@
+Before you start writing any Ember code, it's a good idea to get an overview of how an
```

Comment:
The RFC calls this section "anatomy of an app." Up to you whether you use that name. Feedback suggests that this page, by any name, will be popular!